src/runtime/flags.ts

```diff
@@ -12,12 +12,19 @@
 
 class FlagDefaults {
   static useNewStorageStack = false;
+  /** If true, runs tests flagged as bazel tests. */
+  static bazel = false;
```

Comment:
I don't know if these two make sense here since you are never going to read them from ts code?

Reply:
Thanks! Removed (and added comments as requested.